### PR TITLE
removed duplicate use statement

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: Admin/PostAdmin.php [hunk 1]
```diff
@@ -32,7 +31,7 @@ class PostAdmin extends Admin
     protected $userManager;
 
     /**
-     * @var Pool
+     * @var FormatterPool
```
Comment (Soullivaneuh) [inline Admin/PostAdmin.php]: `Pool` was used. I suggest to remove the *real* unused statement instead: `FormatterPool`.

Comment (OskarStark) [inline Admin/PostAdmin.php]: nope

```FormatterPool``` is used more then three times i think.

scroll down ;-)

```Pool``` just there

Comment (Soullivaneuh) [inline Admin/PostAdmin.php]: I said nothing so. :-D

**Comment by Soullivaneuh** [comment]
Thank you @OskarStark.

**Comment by OskarStark** [comment]
:smile: 

thank you @Soullivaneuh

**Pull request merged**